ops/ccs-ops-misc/load_test/common/bfd_user_base.py

```diff
@@ -1,25 +1,56 @@
 '''Base class for Locust tests run against the FHIR endpoints.
 '''
 
+from typing import Callable, Dict, List, Optional, Union
 import json
 import logging
-import os
-
-from typing import Callable, Dict, List, Union
-from common import config, data, test_setup as setup, validation
-from common.stats.aggregated_stats import StatsCollector
-from common.stats.stats_compare import DEFAULT_DEVIANCE_FAILURE_THRESHOLD, validate_aggregated_stats
-from common.stats.stats_config import StatsStorageType
-from common.stats.stats_loaders import StatsLoader
-from common.stats.stats_writers import StatsJsonFileWriter, StatsJsonS3Writer
-from common.url_path import create_url_path
+import urllib3
+import urllib3.exceptions
 from locust import HttpUser, events
 from locust.env import Environment
+from locust.argument_parser import LocustArgumentParser
+from common import custom_args, data, validation
+from common.locust_utils import is_distributed, is_locust_worker
+from common.stats import stats_compare, stats_writers
+from common.stats.aggregated_stats import StatsCollector
+from common.stats.stats_config import StatsConfiguration
+from common.url_path import create_url_path
 
-import urllib3
+@events.init_command_line_parser.add_listener
+def _(parser: LocustArgumentParser, **kwargs) -> None:
+    custom_args.register_custom_args(parser)
+
+@events.init.add_listener
+def _(environment: Environment, **kwargs) -> None:
+    if is_distributed(environment) and is_locust_worker(environment):
+        return
+    
+    custom_args.adjust_parsed_run_time(environment)
+    validation.setup_failsafe_event(environment)
+    
+@events.quitting.add_listener
+def _(environment: Environment, **kwargs) -> None:
```

Comment:
Unused kwargs and some trailing whitespace issues.
```suggestion
def _(parser: LocustArgumentParser) -> None:
    custom_args.register_custom_args(parser)

@events.init.add_listener
def _(environment: Environment) -> None:
    if is_distributed(environment) and is_locust_worker(environment):
        return

    custom_args.adjust_parsed_run_time(environment)
    validation.setup_failsafe_event(environment)

@events.quitting.add_listener
def _(environment: Environment) -> None:
```